src/components/Link.tsx

```diff
@@ -28,20 +30,18 @@ export default function Link({ meeting, setState, state }: LinkProps) {
     );
   }
 
+  const navigate = () => {
+    searchParams.set('meeting', meeting.slug);
+    setSearchParams(searchParams);
+  };
+
   return (
     <>
       <a
-        href={formatUrl({ ...state.input, meeting: meeting.slug }, settings)}
```

Comment:
i think it would be good to keep the `href` here for SEO reasons. also it makes the cursor be a pointer

Reply:
i see that React Router's Link component does this (i see a `href` on the back to meetings link when it's rendered) which is great. would a RouterLink work here?

Reply:
Updated to use NavLink. Instead of setSearchParams